kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+
+  <div>
+    <ul>
+      <KCard
+        class="card"
+        :to="to"
+        :headingLevel="2"
```

Comment:
**[required]** I'd suggest that `<AccessibleFolderCard>` has a `headingLevel` prop and just passes its value to `KCard` via the same prop. Reason being that `<AccessibleFolderCard>` will be used from more pages in Kolibri, and on each page it can be a member of a different level. This will ensure that devs always make sure to configure it explicitly and correctly in the context of a given page.